cmd/internal/cli/actions_linux.go

```diff
@@ -73,10 +87,9 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
 		uid = uint32(targetUID)
 
 		engineConfig.SetTargetUID(targetUID)
-	} else if uidParam != "" {
-		sylog.Warningf("uid security feature requires root privileges")
-	}
-	if os.Getuid() == 0 && gidParam != "" {
+	})
+
+	checkPrivileges(uidParam != "", "gid security feature", func() {
```

Comment:
I think this should be:

`checkPrivileges(gidParam != "", "gid security feature", func() {`

Reply:
Oh, right, thanks!